src/dswx_sar/save_mgrs_tiles.py

```diff
@@ -1,73 +1,114 @@
 import os
 import time
-import numpy as np
 import mgrs
 import logging
 import mimetypes
-from osgeo import gdal, osr
-from pyproj import CRS
 import h5py
 import glob
-import mgrs
+import ast
+import rasterio
+import numpy as np
+import geopandas as gpd
 import datetime
+
+from osgeo import gdal, osr
+from pyproj import CRS
+from rasterio.warp import transform_bounds
+from shapely.geometry import Polygon
+
 from dswx_sar.version import VERSION as software_version
 from dswx_sar import dswx_sar_util
 from dswx_sar.dswx_runconfig import _get_parser, RunConfig
 
 
 logger = logging.getLogger('dswx_s1')
 
-def get_geographic_boundaries_from_mgrs_tile(mgrs_tile_name, verbose=False):
-    """Get bounding box cooridnate from MGRS tile
+def get_geographic_boundaries_from_mgrs_tile(mgrs_tile_name,
```

Comment:
```suggestion
def get_bounding_box_from_mgrs_tile(mgrs_tile_name,
```
Please, consider updating the name of this function.